README.rst

```diff
@@ -1,5 +1,7 @@
 .. This file contains a series of comments that are used to include sections of this README in other files. Do not modify these comments unless you know what you are doing. tag:intro-begin
 
+.. image:: certbot-logo.svg
```

Comment:
When having GitHub render this file, it shows a broken image. This is true both when checking out the branch on the main page of the repo and when clicking "View" in the top right corner of the PR screen. This to me suggests there might be a problem here.